spec/defines/foreman_proxy_settings_file_spec.rb

```diff
@@ -11,59 +11,66 @@
+              owner: 'root',
+              group: ['FreeBSD', 'DragonFly'].include?(facts[:osfamily]) ? 'foreman_proxy' : 'foreman-proxy',
+              mode: '0640',
+            )
+          end
+
+          it 'should set :enabled in config' do
+            verify_exact_contents(catalogue, config_path, ['---', ':enabled: https'])
+          end
+
+          it { is_expected.not_to contain_foreman_proxy__feature('Test') }
         end
-        it { is_expected.to contain_file(config_path).that_requires('Class[foreman_proxy::install]') }
-        it { is_expected.to contain_file(config_path).that_notifies('Class[foreman_proxy::service]') }
 
-        it 'should set :enabled in config' do
-          verify_exact_contents(catalogue, config_path, ['---', ':enabled: https'])
+        context 'with feature' do
+          let(:params) { { feature: 'Test' } }
+          it { is_expected.to contain_foreman_proxy__feature('Test') }
         end
 
-        it { is_expected.not_to contain_foreman_proxy__feature('Test') }
-      end
+        context 'with listen_on => both' do
+          let(:params) { { listen_on: 'both' } }
 
-      context 'with feature' do
-        let(:params) { { feature: 'Test' } }
-        it { is_expected.to contain_foreman_proxy__feature('Test') }
-      end
+          it 'should set :enabled to true in config' do
+            verify_exact_contents(catalogue, config_path, ['---', ':enabled: true'])
+          end
+        end
 
-      context 'with listen_on => both' do
-        let(:params) { { listen_on: 'both' } }
+        context 'with listen_on => http' do
+          let(:params) { { listen_on: 'http' } }
 
-        it 'should set :enabled to true in config' do
-          verify_exact_contents(catalogue, config_path, ['---', ':enabled: true'])
+          it 'should set :enabled to http in config' do
+            verify_exact_contents(catalogue, config_path, ['---', ':enabled: http'])
+          end
         end
-      end
 
-      context 'with listen_on => http' do
-        let(:params) { { listen_on: 'http' } }
+        context 'with enabled => false' do
+          let(:params) { { enabled: false } }
+
+          it 'should set :enabled to false in config' do
+            verify_exact_contents(catalogue, config_path, ['---', ':enabled: false'])
+          end
 
-        it 'should set :enabled to http in config' do
-          verify_exact_contents(catalogue, config_path, ['---', ':enabled: http'])
+          context 'with feature' do
+            let(:params) { { enabled: false, feature: 'Test' } }
+            it { is_expected.not_to contain_foreman_proxy__feature('Test') }
+          end
         end
       end
 
-      context 'with enabled => false' do
-        let(:params) { { enabled: false } }
-
-        it 'should set :enabled to false in config' do
-          verify_exact_contents(catalogue, config_path, ['---', ':enabled: false'])
-        end
+      context 'with foreman included' do
```

Comment:
```suggestion
      context 'with foreman_proxy included' do
```

Reply:
I actually came to the exact same conclusion just now.